cli/providers/destroy.go

```diff
@@ -6,13 +6,14 @@ import (
 	"github.com/pchico83/i2kit/cli/providers/aws"
 	"github.com/pchico83/i2kit/cli/schemas/environment"
 	"github.com/pchico83/i2kit/cli/schemas/service"
+	"github.com/sirupsen/logrus"
 )
 
 //Destroy destroys a given service in a given environment
-func Destroy(s *service.Service, e *environment.Environment) error {
+func Destroy(s *service.Service, e *environment.Environment, log *logrus.Logger) error {
```

Comment:
Can you test and see if an external program consuming i2kit can use this function as you intend it? We tried the same pattern with types from the aws library and it didn't work. We might need to define an i2kit-level interface instead (one that logrus.Logger meets).